lib/cequel/metal/keyspace.rb

```diff
@@ -266,6 +287,7 @@ def client_without_keyspace
       def client_options
         {hosts: hosts, port: port}.tap do |options|
           options.merge!(credentials) if credentials
+          options[:server_cert] = server_cert if server_cert
```

Comment:
Not quite true, you could have many different combinations. You could have only ssl: true, only server_cert, or all of server_cert, client_cert, private_key, and passphrase.... I have changed the extraction to place it all in a single hash that gets merged into the config if all values aren't nil. http://datastax.github.io/ruby-driver/features/security/ssl_encryption/ details the config options one could pass to Cassandra.cluster